app/src/main/java/org/gdg/frisbee/android/app/App.java

```diff
@@ -127,9 +131,46 @@ public void onCreate() {
         mLocationFinder = new GingerbreadLastLocationFinder(this);
         mLocationFinder.setChangedLocationListener(this);
         updateLastLocation();
+        
+        //Init TaggedEventSeries
+        mTaggedEventSeriesList = new ArrayList<>();
+        addTaggedEventSeriesIfDateFits(new TaggedEventSeries("wtm",
+                R.drawable.drw_ic_wtm,
+                R.string.wtm,
+                R.string.wtm_description,
+                R.drawable.ic_wtm,
+                Const.START_TIME_WTM,
+                Const.END_TIME_WTM));
+        addTaggedEventSeriesIfDateFits(new TaggedEventSeries("studyjam",
+                R.drawable.drw_ic_studyjams,
+                R.string.studyjams,
+                R.string.studyjams_description,
+                R.drawable.ic_studyjams,
+                Const.START_TIME_STUDY_JAMS,
+                Const.END_TIME_STUDY_JAMS));
+        addTaggedEventSeriesIfDateFits(new TaggedEventSeries("io-extended",
+                R.drawable.drw_ic_ioextended,
+                R.string.ioextended,
+                R.string.ioextended_description,
+                R.drawable.ic_ioextended,
+                Const.START_TIME_IOEXTENDED,
+                Const.END_TIME_IOEXTENDED));
     }
 
-    public void migrate(int oldVersion, int newVersion) {
+    /**

```

Comment:
This kind of Javadoc is useless because it's very redundant with the code, specially since it's a private method. You can remove it.